mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(num) {
+  var netValue =(num*20)/100 + num;
```

Comment:
This is an interesting solution and it works fully.

I would usually advise that you specify things like the tax rate as a constant, in more complicated functions, you may reference this tax rate multiple times. An example of this using your original calculation would be:

```javascript
function calculateSalesTax(num) {
    const SALES_TAX = 20;
    
    return ((num * SALES_TAX) / 100) + num;
}
```